Source/Core/Core/PowerPC/JitCommon/JitCache.h

```diff
@@ -17,17 +17,11 @@
 
 class JitBase;
 
-// A JitBlock is block of compiled code which corresponds to the PowerPC
-// code at a given address.
-//
-// The notion of the address of a block is a bit complicated because of the
-// way address translation works, but basically it's the combination of an
-// effective address, the address translation bits in MSR, and the physical
-// address.
-struct JitBlock
+// A JitBlockData is data block of compiled code, which has to be in a
+// fixed layout so that JIT code is able to access the fields.
+// All the fields has to be trival types.
```

Comment:
This should really just be saying that the struct itself needs to be standard layout. Types within it can be non-trivial and offsetof will work completely fine.